README.md

```diff
@@ -163,6 +163,37 @@ does not work it may be because you don't have the correct permissions, and
 need to use `sudo make deploy`.
 See the README.md file in the ports/stm32/ directory for further details.
 
+Building under Windows
+----------------------
+
+MicroPython is built using gnu make and requires various gnu utilities ported
+to Windows. MSYS2 should be installed to provide these gnu utilities.
+
+To use frozen bytecode (.mpy modules) you must build mpy-cross for your host OS.
+This requires mingw32 gcc and it dependancies.
```

Comment:
dependancies -> dependencies. Also, the official name is mingw-w64 so better stick to that: Micropython does not support mingw (also named mingw32) anymore.

Reply:
ok